client/app/dedicated/server/bandwidth/private-order/private-order.routing.js

```diff
@@ -0,0 +1,25 @@
+export default /* @ngInject */ ($stateProvider) => {
+  // Parent states where the modal could be called
+  const parentStates = [
+    'app.dedicated.server.dashboard',
+    'app.dedicated.server.interfaces',
```

Comment:
Imo, you should have two modules with two separate routing (`dedicatedServerDashboard` and `dedicatedServerInterfaces`) calling the same component (in a third module `privateBandwidthorder` for instance). We can talk about it if needed :)

Reply:
Yes, I'd like too. I don't see what's really needed here :)